scripts/nuget.fsx

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env fsharpi
+
+open System
+open System.IO
+open System.Linq
+
+#r "System.Configuration"
+#load "../fsx/InfraLib/Misc.fs"
+#load "../fsx/InfraLib/Process.fs"
+#load "../fsx/InfraLib/Network.fs"
+open FSX.Infrastructure
+open Process
+open System.IO
+open System.Linq
+
+let MaybeUpload (packageName: string) =
+    let outputDir = Path.Combine(Directory.GetCurrentDirectory(), "Output") |> DirectoryInfo
+    let nupkgFile = outputDir.EnumerateFiles().Single(fun file -> file.FullName.EndsWith ".nupkg")
```

Comment:
just realized that packageName parameter is not being used, maybe we should change this fun to be `fun file -> file.FullName.StartsWith packageName && file.FullName.EndsWith ".nupkg" (this maybe means that you need to change the call to this function from "Xamarin.Essentials" to "DotNetEssentials", I'm not sure

Reply:
So, that code does not seem to work, it doesn't seem to find any file. But I am unable to understand why. I have tried several combinations and I couldn't make it work, passing a void string `""` works, but using other beginning strings of files that actually exist in my setup fail. Maybe I should just remove the packageName variable?

Reply:
use something like this to know what's happening:
```
for file in output.EnumerateFiles() do
    Console.WriteLine ("file:" + file.FullName)
```

Reply:
`file.FullName.StartsWith packageName` was matching the full path not the file name. Fixed.